jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/DeployServerlessApplicationAction.kt

```diff
@@ -75,14 +78,26 @@ class DeployServerlessApplicationAction : AnActionWrapper(
         saveSettings(project, templateFile, stackDialog)
 
         val stackName = stackDialog.stackName
+        val stackId = stackDialog.stackId
+
+        if (stackId == null) {
+            continueDeployment(project, stackName, templateFile, stackDialog)
+        } else {
+            warnResourceOperationAgainstCodePipeline(project, stackName, stackId, ResourceType.CLOUDFORMATION_STACK, Operation.DEPLOY) {
+                continueDeployment(project, stackName, templateFile, stackDialog)
+            }
+        }
+    }
+
+    private fun continueDeployment(project: Project, stackName: String, templateFile: VirtualFile, stackDialog: DeployServerlessApplicationDialog) {
         val deployDialog = SamDeployDialog(
```

Comment:
Indent size 4, continuation indent 8 are the kotlin standards

Reply:
This is following the same (indent size 4, continuation indent 8) kotlin standards. What is the ask here?

Reply:
It looks to be 8, you have changed the formatting in several areas of the code base and I don't see anything that causes it to be wrong in the first place

Reply:
Actually it is supposed to be 4 according to yole

Reply:
https://github.com/JetBrains/kotlin/blob/189260c0134741a7e1fe2783193b34c8a241901a/.idea/codeStyles/Project.xml#L24

Reply:
I changed Kotlin code style in my IDE: Editor > Code Style > Kotlin > Set from... > Predefined Style > Kotlin style guide. It uses the indent size 4 after parenthesis now.